pyproject.toml

```diff
@@ -2,12 +2,13 @@
 requires = ["setuptools>=61.0.0", "wheel"]
 build-backend = "setuptools.build_meta"
 
-[tool.setuptools]
-packages = ["xl2times"]
+[tool.setuptools.packages.find]
+where = [""]
+include = ["xl2times", "xl2times.*"]
 
 [project]
 name = "xl2times"
-version = "0.1.0"
+version = "0.1.0-alpha.2"
```

Comment:
I'm bumping the version number to be able to test the publish action when I make this alpha pre-release (after merging this PR). In my next PR I'll set it back to 0.1.0 and publish to the real PyPI.